etc/ReplayOfflineConfiguration.py

```diff
@@ -632,7 +638,7 @@
                do_reco=True,
                write_dqm=True,
                alca_producers=["HcalCalNoise"],
-               dqm_sequences=["@common", "@jetmet", "@hcal"],
+               dqm_sequences=["@common", "@jetmet", "@L1TMon", "@hcal"],
```

Comment:
@jfernan2 It looks like the replays is indeed ading DQM sequences to the MET dataset. Is this a mistake?